### PR TITLE
Publish slides

### PR DESCRIPTION
**Pull request by ffevotte**
This commit publishes the slides of yesterday's meetup (as well as the necessary markup to display the links to download them).

Feel free to change the styling as needed

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -32,12 +32,29 @@ table tr:last-child p {
...
 
+.summary, .slides {
+    padding-left: 2em;
+    vertical-align: middle;
+}
+
+.slides a:before {
+    display: inline-block;
+    width: 1.8em;
+    height: 2em;
```
Comment (mfherbst) [inline main.css]: For my taste the pdf logo is a little too large.

**Review by mfherbst — COMMENTED** [review]
Looks generally good to me. Thanks.

File: main.css (hunk 1 above)
Comment (ffevotte) [reply main.css]: I pushed a new commit introducing a smaller logo. Is it better like this?

Comment (mfherbst) [reply main.css]: LGTM. Thanks.

**Review by mfherbst — APPROVED** [review]

**Comment by frapac** [comment]
Thanks for the PR!

**Pull request merged**